betty/locale/localizable/__init__.py

```diff
@@ -329,31 +332,91 @@ def localize(self, localizer: Localizer, /) -> Localized & str:
         return LocalizedStr(self._text, locale=self._locale)
 
 
+ShorthandCountableStaticTranslations: TypeAlias = Mapping[LocaleLike, Mapping[str, str]]
+CountableStaticTranslationsMapping: TypeAlias = Mapping[Locale, Mapping[str, str]]
+
+
 @final
-class CountablePlain(CountableLocalizable):
-    """
-    Turn plain strings into a :py:class:`betty.locale.localizable.CountableLocalizable` without any actual translations.
-    """
+class CountableStaticTranslations(CountableLocalizable):
+    """
+    A countable localizable backed by static translations.
+    """
+
+    _translations: CountableStaticTranslationsMapping
+
+    def __init__(self, translations: ShorthandCountableStaticTranslations, /):
+        self._translations = {
+            self._ensure_locale(locale, locale_translations): locale_translations
+            for locale, locale_translations in translations.items()
+        }
+
+    def _ensure_locale(
+        self, locale: LocaleLike, translations: Mapping[str, str]
+    ) -> Locale:
+        locale = ensure_locale(locale)
+        provided_plural_tags = set(translations.keys())
+        locale_plural_tags = {
+            *locale.plural_form.tags,
+            # "other" is always required (https://cldr.unicode.org/index/cldr-spec/plural-rules) but not always returned
```

Comment:
Hardcode localizable descriptions of the tags and link to the CLDR docs in the error.